packages/gitbook/src/components/PageAside/PageAside.tsx

```diff
@@ -65,135 +67,144 @@ export function PageAside(props: {
                 withHeaderOffset ? 'lg:max-h-[calc(100vh_-_4rem)]' : 'lg:max-h-[100vh]',
                 withHeaderOffset ? 'top-16' : 'top-0',
                 'h-[100vh]',
-
-                // When in api page mode, we display it as an overlay on non-large resolutions
-                'page-api-block:xl:max-2xl:backdrop-blur-md',
-                'page-api-block:xl:max-2xl:fixed',
-                'page-api-block:xl:max-2xl:right-8',
+                'page-api-block:xl:max-2xl:z-10',
                 'page-api-block:xl:max-2xl:w-56',
-                'page-api-block:xl:max-2xl:bg-light-2/9',
                 'page-api-block:xl:max-2xl:rounded',
                 'page-api-block:xl:max-2xl:h-auto',
                 'page-api-block:xl:max-2xl:py-0',
                 'page-api-block:xl:max-2xl:mt-3',
-                'dark:page-api-block:xl:max-2xl:bg-dark-2/8',
                 withHeaderOffset
-                    ? 'page-api-block:xl:max-2xl:top-16'
+                    ? 'page-api-block:xl:max-2xl:top-20'
```

Comment:
is `top-20` also applicable to non-sections site?

Reply:
Changed the approach.